### DISCUSSION TITLE
django.db.utils.ProgrammingError: relation "silk_request" does not exist

### DISCUSSION THREAD

**Discussion by edsonblwolf** [amidaware/tacticalrmm#1611] [General]
ran troubleshoot_server.sh
```

Getting summary output of logs
  File "/rmm/api/env/lib/python3.11/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/rmm/api/env/lib/python3.11/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/rmm/api/env/lib/python3.11/site-packages/psycopg/cursor.py", line 737, in execute
    raise ex.with_traceback(None)
django.db.utils.ProgrammingError: relation "silk_request" does not exist
LINE 1: INSERT INTO "silk_request" ("id", "path", "query_params", "r...

```

I installed the packages:
pip install django_extensions django-silk

And still the error appears

**Comment by wh1te909** [comment]
are you running with DEBUG = True?

**Reply by edsonblwolf** [reply]
Yes, only this information
[screen](https://imgur.com/7ahA2Sp.png)

**Reply by wh1te909** [reply]
why are you running with DEBUG?

**Reply by silversword411** [reply]
![2023-08-23_092034 - chrome_Troubleshooting_-_Tactical_RMM_Documentation_-_Goo](https://github.com/amidaware/tacticalrmm/assets/13395348/4d9ae05b-8a5b-4c06-b060-5b1ecd615042)

?

**Reply by edsonblwolf** [reply]
To try to find the problem

**Reply by edsonblwolf** [reply]
Basically if I pay it works. haha

**Reply by wh1te909** [reply]
what? look at the screenshot @silversword411 posted. this is usually a time sync issue.

**Reply by silversword411** [reply]
> Basically if I pay it works. haha

Huh?

Follow the big red numbers. 
Do Each Step. 
Read. 
Follow instructions.
